src/Data/Vec/Recursive.agda

```diff
@@ -17,17 +17,23 @@
 module Data.Vec.Recursive where
 
 open import Level using (Level; lift)
+open import Function.Bundles using (mk↔′)
+open import Function.Properties.Inverse using (↔-isEquivalence; ↔refl)
 open import Data.Nat.Base as Nat using (ℕ; zero; suc)
 open import Data.Empty.Polymorphic
 open import Data.Fin.Base as Fin using (Fin; zero; suc)
 open import Data.Product as Prod using (_×_; _,_; proj₁; proj₂)
+open import Data.Product.Algebra using (×-cong)
 open import Data.Sum.Base as Sum using (_⊎_)
 open import Data.Unit.Base using (tt)
 open import Data.Unit.Polymorphic.Base using (⊤)
 open import Data.Vec.Base as Vec using (Vec; _∷_)
+open import Data.Vec.N-ary using (N-ary)
 open import Function
 open import Relation.Unary
-open import Agda.Builtin.Equality using (_≡_)
+open import Agda.Builtin.Equality using (_≡_; refl)
```

Comment:
This import can be removed in favour of the `Relation.Binary.PropositionalEquality.Core` immediately after?

Reply:
I removed this line.